appscale/tools/appscale_tools.py

```diff
@@ -273,7 +273,8 @@ def gather_logs(cls, options):
       '/var/log/monit.log*',
       '/var/log/nginx',
       '/var/log/syslog*',
-      '/var/log/zookeeper'
+      '/var/log/zookeeper',
+      '/opt/cassandra/cassandra/logs/system.log'
```

Comment:
Would it be worhtwhile to get all the files in that directory? Or system.log is enough?